packages/psss-server/src/connections/ReportConnection.ts

```diff
@@ -37,13 +37,19 @@ export class ReportConnection extends ApiConnection {
     if (!orgUnitCodes) {
       throw new Error('No organisationUnitCodes provided');
     }
+
+    const [startWeek, endWeek] = periods;
+    const [startDate] = convertPeriodStringToDateRange(startWeek);
+    const [, endDate] = convertPeriodStringToDateRange(endWeek);
+
     if (orgUnitCodes.length === 0) {
       return buildEmptyReport(periods);
     }
```

Comment:
A very minor edit, but I think the conversion of periods variable to startDate and endDate should come after checking that there are no orgUnitCodes, as there isn't any point running that code if orgUnitCodes length is 0.